db/schema.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 # This file is auto-generated from the current state of the database. Instead
```

Comment:
> This file is auto-generated

how about we exclude this one from the robogarda then?  would be more than annoying to always have to format this file too considering it's auto-generated

Reply:
Reverted the `schema.rb` changes since the file is already excluded anyway (oops).